ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -127,8 +127,8 @@
 
 ### BGP Neighbors
 
-| Neighbor | Remote AS | VRF | Send-community | Maximum-routes | Allowas-in |
-| -------- | --------- | --- | -------------- | -------------- | ---------- |
+| Neighbor | Remote AS | VRF | Send-community | Maximum-routes | Allowas-in | Bfd |
```

Comment:
The heading should be `BFD`

Reply:
I will fix it to avoid further delay

Reply:
Sorry, I force pushed and your changes didn't take off.

Reply:
Changing it to BFD now. There is one more Bfd in the doc, changing it as well.

Reply:
Updated the required changes.